### PR TITLE
clap-host 0.2 (new formula)

### PR DESCRIPTION
**Pull request by abique**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Hi,

Regarding the source repository popularity, please understand that this application is a part of a larger project: https://github.com/free-audio/clap

Thank you very much,
Alexandre

### CODE REVIEW

**Comment by carlocab** [comment]
Thanks for the PR.

It might be part of a larger project, but it is also a distinct piece of software from https://github.com/free-audio/clap. We could/should probably package that one separately. If there's enough interest in your example host we could potentially package this in the future too -- but we shouldn't be bundling them together by pulling in your library as a submodule.

If you wish, you can host this formula in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

File: Formula/clap-host.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  depends_on "rtaudio"
+  depends_on "rtmidi"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", "-DCLAP_HOST_BUNDLE=FALSE", *std_cmake_args
+    system "cmake", "--build", "build", "--target", "install"
+  end
+
+  test do
+    system "true"
```
Comment (carlocab) [inline Formula/clap-host.rb:21]: If you choose to submit this formula again in the future when your host is more popular, please include a proper test.

**Comment by abique** [comment]
> It might be part of a larger project, but it is also a distinct piece of software from https://github.com/free-audio/clap. We could/should probably package that one separately. If there's enough interest in your example host we could potentially package this in the future too -- but we shouldn't be bundling them together by pulling in your library as a submodule.

CLAP is a plugin standard, and clap-host is the reference plugin host.
So anyone who wants to get started with CLAP will probably want to checkout the clap-host and use it to test and validate a plugin under development.

Yes, we could make a formula for CLAP, but as this one is headers only, I think it is more likely that devs just add it as a submodule to their own projects.

File: Formula/clap-host.rb (hunk 1 above)
Comment (abique) [reply Formula/clap-host.rb:21]: Thank you, I'll take care of that.

Comment (abique) [reply Formula/clap-host.rb:21]: It has been addressed now.

**Pull request closed without merging**